packages/node/edge/index.ts

```diff
@@ -17,14 +17,13 @@ export default class LogtoClient extends BaseClient {
           ? async (...args: Parameters<typeof fetch>) => {
               const [input, init] = args;
 
+              // Encode to base64 using btoa
+              const base64Credentials = btoa(`${config.appId}:${config.appSecret ?? ''}`);
```

Comment:
nit: use `window.btoa` to workaround the deprecated mark?

Reply:
in Node.js, `window` is not available, however, `globalThis` is OK